luminoth/tools/dataset/readers/object_detection/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .object_detection_reader import ObjectDetectionReader  # noqa
+
+from .coco import COCOReader  # noqa
+from .csv_reader import CSVReader  # noqa
+from .flat_reader import FlatReader  # noqa
```

Comment:
Standardize: remove `_reader` from filenames. We already know they are readers because they are inside `dataset.readers`.

Reply:
We don't want to have a `csv` module. I considered `flat` to be too generic as well.

Reply:
Worried about clashes? I am not sure its *that* bad here.